### ISSUE TITLE
Agent with Ollama can't find OllamaLanguageModelInstruction api.

### ISSUE THREAD

**Issue by acer1204** [tryAGI/LangChain#365]
### Describe the bug

after install this two package
1.LangChain.Core
2.LangChain.Providers.Ollama

copy the example code for ollama, but it's show OllamaLanguageModelInstruction and OllamaLanguageModelOptions not found.

### Steps to reproduce the bug

1.install LangChain.Core
2.LangChain.Providers.Ollama
3.copy agent example for ollama

### Expected behavior

it's can run

### NuGet package version

LangChain v0.15.0
LangChain.Core v0.15.0
LangChain.Providers.Ollama v0.15.0

**Comment by quannv108** [comment]
Try use this
```csharp
var provider = new OllamaProvider(
            "http://localhost:11434",
            options: new RequestOptions
            {
                Temperature = 0,
            });
        var model = new OllamaChatModel(provider, id: "llama3").UseConsoleForDebug();
```
look like the wiki page is outdate and need to be update

**Comment by HavenDV** [comment]
Yes, it's best to use these tests as an example, we are in the process of migrating documentation from wiki to mkdocs
https://github.com/tryAGI/LangChain/blob/main/src/Meta/test/WikiTests.AgentWithOllama.cs

**Comment by HavenDV** [comment]
https://tryagi.github.io/LangChain/wiki/AgentWithOllama/
